index.js

```diff
@@ -49,7 +69,16 @@ module.exports = (babel, options) => {
     presets: [
       [
         require('@babel/preset-env').default,
-        {include, loose, modules, targets, bugfixes: true, corejs: 3, useBuiltIns: 'entry'},
+        {
+          include,
+          loose,
+          modules,
+          targets,
+          bugfixes: true,
+          corejs: 3,
+          useBuiltIns: 'entry',
+          browserslistEnv: env.includes('modern') ? 'production-modern' : undefined, // defaults to [production]
```

Comment:
Yes, since we want development to be able to output code for both modes that production will run in. But this naming is not the best; maybe we can rename the browser group to `modern-group` or just `modern`